specs/payment-api.yaml

```diff
@@ -767,56 +626,41 @@ paths:
         200:
           $ref: '#/components/responses/OK'
         401:
-          $ref: '#/components/responses/Unauthorized'
+          $ref: 'common.yaml#/responses/Unauthorized'
         404:
-          $ref: '#/components/responses/NotFound'
+          $ref: 'common.yaml#/responses/NotFound'
         500:
-          $ref: '#/components/responses/ServerError'
+          $ref: 'common.yaml#/responses/ServerError'
 
-  /requestor/payments:
+  /payments:
     get:
       tags:
         - requestor
-      summary: Get outgoing Payments.
-      operationId: getOutgoingPayments
+        - provider
+      summary: Get Payments.
+      operationId: getPayments
       description: >
         Payments can be treated as events and this method can be used to listen
         for new payments by long-polling.  If there are any payments the method
         will return them immediately. If there are none the method will wait
         until one appears or timeout passes.
-        `laterThan` parameter can be used in order to get just the 'new'
+        `afterTimestamp` parameter can be used in order to get just the 'new'
         payments. Setting the parameter value to the timestamp of the last
         processed payment ensures that no payments will go unnoticed.
       parameters:
-        - $ref: '#/components/parameters/eventTimeout'
-        - $ref: '#/components/parameters/laterThan'
+        - $ref: 'common.yaml#/parameters/pollTimeout'
+        - $ref: 'common.yaml#/parameters/afterTimestamp'
+        - $ref: 'common.yaml#/parameters/maxEvents'
+        - $ref: 'common.yaml#/parameters/appSessionId'
       responses:
         200:
           $ref: '#/components/responses/PaymentList'
         401:
-          $ref: '#/components/responses/Unauthorized'
+          $ref: 'common.yaml#/responses/Unauthorized'
         500:
-          $ref: '#/components/responses/ServerError'
+          $ref: 'common.yaml#/responses/ServerError'
 
-  /requestor/payments/{paymentId}:
-    get:
-      tags:
-        - requestor
-      summary: Get outgoing Payment.
-      operationId: getOutgoingPayment
-      parameters:
-        - $ref: '#/components/parameters/paymentId'
-      responses:
-        200:
-          $ref: '#/components/responses/Payment'
-        401:
-          $ref: '#/components/responses/Unauthorized'
-        404:
-          $ref: '#/components/responses/NotFound'
-        500:
-          $ref: '#/components/responses/ServerError'
-
-  /requestor/accounts:
+  /sendAccounts:
```

Comment:
misleading name in our (market team) opinion

Reply:
Any proposed alternative?

Reply:
```suggestion
  /requestorAccounts:
```

Reply:
Makes sense, thank you. Agreed.

Reply:
👍